src/lib/clean_number.js

```diff
@@ -13,17 +13,16 @@ var isNumeric = require('fast-isnumeric');
 
 var BADNUM = require('../constants/numerical').BADNUM;
 
-// precompile these regex's for speed
-var FRONTJUNK = /^['"%,$#\s']+/;
-var ENDJUNK = /['"%,$#\s']+$/;
+// precompile for speed
+var JUNK = /['"%,$#\s]/g;
```

Comment:
Back to what it was pre #1078 on

 https://github.com/plotly/plotly.js/pull/1078/files#diff-9664b6a57e7771275ffd0c8805ea5917L30